test-app/ember-cli-build.js

```diff
@@ -6,6 +6,7 @@ module.exports = async function (defaults) {
       setConfig: {
         'ember-data-factory-guy': {
           useStringIdsOnly: true,
+          interceptor: process.env.INTERCEPTOR || 'pretender',
```

Comment:
this shows how you can configure which interceptor to use application wide

We have ember-try modify this so that we can run the same test suite against both Pretender and MSW